src/styles/components/_audio-page.sass

```diff
@@ -240,6 +240,14 @@
   .uploading__container .row
     margin-bottom: 0
 
+  .playwave__container
+    margin-bottom: 0
+
+  .waveform__timestamp
+    color: $gray-color
+    text-align: right
+    font-size: 0.7rem
```

Comment:
It's completely up to you, but have you considered not using REMs?
https://mindtheshift.wordpress.com/2015/04/02/r-i-p-rem-viva-css-reference-pixel/

Pretty much depends on personal preference – I've ditched those units entirely.

Reply:
Oh no, I feel into that rabbit hole just from you mentioning that. Lots of opinions on this on the internet. I'm gonna defer final decision on that for when @tinykb goes thru w/ a design eye and we get some specs about font size options.  I'm hoping to just have a handful of sizes to choose from "largest, larger, large, medium, small, smaller, smallest"